webpack.config.js

```diff
@@ -1,17 +1,39 @@
-const HtmlWebpackPlugin = require('html-webpack-plugin'); // eslint-disable-line import/no-unresolved
-
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+// eslint-disable-line import/no-unresolved
 module.exports = {
+  mode: 'development',
```

Comment:
I have noticed you have disabled eslint in your web pack.config file. It is against micro verse policy to disable linters because linters help us write clean code. It would be nice if you enabled eslint.